exporters/jaeger/src/jaeger_exporter.cc

```diff
@@ -91,10 +94,17 @@ void JaegerExporter::InitializeEndpoint()
 
 bool JaegerExporter::Shutdown(std::chrono::microseconds timeout) noexcept
 {
+  const std::lock_guard<opentelemetry::common::SpinLockMutex> locked(lock_);
   is_shutdown_ = true;
   return true;
 }
 
+const bool JaegerExporter::isShutdown() const noexcept
```

Comment:
No need to return const as the returned value is always a copy?

Reply:
yeah thanks, no need for `const`.